### ISSUE TITLE
could not deploy registry pod 

### ISSUE THREAD

**Issue by priyanka5** [openshift/origin#11770]
I am installing origin 1.3.0 on centos 7.1 using ansible, but keep getting this in registry and router pod:

error: couldn't get deployment docker-registry-1: Get https://172.30.0.1:443/api/v1/namespaces/default/replicationcontrollers/docker-registry-1: dial tcp 172.30.0.1:443: ockopt: no route to host
 

##### Version
oc v1.3.1
kubernetes v1.3.0+52492b4
features: Basic-Auth GSSAPI Kerberos SPNEGO

Server https://ip-172-31-37-255.ec2.internal:8443
openshift v1.3.1
kubernetes v1.3.0+52492b4

##### Steps To Reproduce
NetworkManager is installed and I checked DNS is also working file, any help?

**Comment by knobunc** [comment]
It looks like you kubernetes service is not working correctly.

Please run the debug.sh script referenced here:
  https://docs.openshift.org/latest/admin_guide/sdn_troubleshooting.html#further-help

And get me the resulting tar.

**Comment by priyanka5** [comment]
Hi @knobunc sorry for late reply . Please see the attached tar

**Comment by knobunc** [comment]
bbennett@redhat.com  Thanks

**Comment by priyanka5** [comment]
@knobunc -  have send an email

**Comment by xiangpengzhao** [comment]
@priyanka5 Has this issue been solved? I remembered that I once met this on my OSE3.2 env days ago. I stopped the firewalld on master/nodes and then solved the problem. 
cc @knobunc

**Comment by priyanka5** [comment]
@xiangpengzhao no, I still get the same issue. I remember I tried disabling firewalld but problem still existed. Let me try again to be sure.

**Comment by scheng1** [comment]
Has this issue been solved?

**Comment by scheng1** [comment]
Has this issue been solved? @priyanka5

**Comment by priyanka5** [comment]
@scheng1 yes long back :)

**Comment by scheng1** [comment]
How to solve it ？I have also get the same problem，Thanks @priyanka5

**Comment by priyanka5** [comment]
@scheng1  I am using latest version now, following  documents and it gives no error. Make sure NetworkManager is installed on VMs

**Comment by scheng1** [comment]
@priyanka5 Thanks
